docker-compose.yaml

```diff
@@ -1,27 +1,26 @@
-version: "2.2"
+version: "3"
 services:
   api:
-    image: camptocamp/swissgeol_local_api:latest
     init: true # handle kill signals for rust
+    image: camptocamp/swissgeol_local_api:latest
     ports:
       - 8480:3000
     environment:
       AWS_HOST: http://minio:9000
       AWS_ACCESS_KEY_ID: &minio_user minio
       AWS_SECRET_ACCESS_KEY: &minio_pass minio123
-      ENVIRONMENT_NAME: local-dev
       PGUSER: &db_user www-data
       PGPASSWORD: &db_pass www-data
       PGDATABASE: &db_name swissgeol-local
       PGHOST: db
       PGPORT: "5432"
       APP_PORT: "3000"
-      SQLX_OFFLINE: "true"
+      DATABASE_URL: "postgres://www-data:www-data@db:5432/swissgeol-local"
     command: ["cargo", "watch", "--shell", "cargo run --offline --target x86_64-unknown-linux-musl"]
```

Comment:
This needs the `--poll` flag

Reply:
Also running this in offline mode requires to restart the composition on any dependency change.